openbb_terminal/stocks/options/options_controller.py

```diff
@@ -1463,3 +1473,31 @@ def call_hedge(self, _):
     def call_screen(self, _):
         """Process screen command"""
         self.queue = screener_controller.ScreenerController(self.queue).menu()
+
+    @log_start_end(log=logger)
+    def call_eodchain(self, other_args: List[str]):
+        """Process greeks command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="eodchain",
+            description="Get full option chain at a given end of day.",
+        )
+        parser.add_argument(
+            "-d",
+            "--date",
+            type=valid_date,
+            dest="date",
+            help="Date to get option chain for.  Format YYYY-MM-DD",
+        )
+
+        ns_parser = self.parse_known_args_and_warn(parser, other_args)
+
+        if ns_parser:
+            if not self.ticker:
+                console.print("No ticker loaded. First use `load <ticker>`")
+                return
+            # TODO: Make all models return self.data
```

Comment:
Can probably be done differently after the refactor --> we'll be loading always the full option chain